.gitattributes

```diff
@@ -0,0 +1 @@
+*	-text
```

Comment:
I guess for this repository this is ok, however this change as is replicates what AppVeyor is doing; http://help.appveyor.com/discussions/problems/671-coreautocrlf.